cfgs/robustness_analysis/pretraining_experiment_eros/res001/posttrain_0.toml

```diff
@@ -0,0 +1,82 @@
+name = "posttrain-eros-001-1000it"
+cuda_devices = "0"
+plotting_points = 2500
+resolution = "100%"
+pretrained_model = [
+    "./results/pretrain-eros-res001/eros/polyhedral/it-id-0000/best_model.mdl",
```

Comment:
shouldn't the pretrained model be the same for all? 🤔

Reply:
I trained for the ground-truth-resolution comparison multiple models. Thus, why not train with multiple "starting" points if available in any case.

Reply:
Basically, getting the randomness from different starting points (generated from different seeds) rather than different seeds in the fine-tuning.

Even though, I also had some experiments with different seeds in the fine-tuning with similar results. But including the "seed" parameter (for all experiments) in fine-tuning and pertaining would have increased the number of training runs from X to X^2.

Reply:
I see, makes sense :v: